runtime/acala/src/lib.rs

```diff
@@ -910,23 +916,20 @@ impl orml_vesting::Config for Runtime {
 }
 
 parameter_types! {
-	pub MaximumSchedulerWeight: Weight = Perbill::from_percent(10) * RuntimeBlockWeights::get().max_block;
-	// Retry a scheduled item every 25 blocks (5 minute) until the preimage exists.
-	pub const NoPreimagePostponement: Option<u32> = Some(5 * MINUTES);
+	pub MaximumSchedulerWeight: Weight = Perbill::from_percent(80) * RuntimeBlockWeights::get().max_block;
```

Comment:
@xlc is 10%->80% intentional

Reply:
https://github.com/paritytech/polkadot/blob/1b1bc62e4447ef1d444869c2fc821c70d02d1d00/runtime/polkadot/src/lib.rs#L178
Polkadot and Kusama are 80%.  The original parameter was causing a lot of failures in the integration tests